### PR TITLE
Add test to verify Dashboard detects OpenShift Pipelines operator

### PR DESCRIPTION
**Pull request by Raghul-M**
Automation of ODS-2274.

### CODE REVIEW

File: poetry.lock
Comment (jstourac) [inline poetry.lock:1]: I don't think this file should be part of this PR? :slightly_smiling_face:

File: ods_ci/run_robot_test.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ EMAIL_SERVER_SSL=false
 EMAIL_SERVER_UNSECURE=false
 OPEN_REPORT_IN_BROWSER=false
 REPORT_BROWSER="firefox" # Default browser to open reports in
-SUBFOLDER=false
+SUBFOLDER=true
```
Comment (jgarciao) [inline ods_ci/run_robot_test.sh:25]: This change should not be in the PR

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource [hunk 2]
```diff
@@ -26,6 +26,8 @@ Open Data Science Projects Home Page
     Wait Until Page Contains    View your existing projects or create new projects.    timeout=30
     Maybe Wait For Dashboard Loading Spinner Page
 
+
```
Comment (jgarciao) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource:29]: It should be only one line between keywords

File: ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/430__data-science-pipelines-operator-dependency.robot [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Suite Setup        Pipelines Suite Setup
+Suite Teardown     Pipelines Suite Teardown
+
+
+*** Variables ***
+
+
+*** Test Cases ***
+Verify Dashboard Disables Pipelines When OpenShift Pipelines Operator Is Not Installed
```
Comment (jgarciao) [inline ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/430__data-science-pipelines-operator-dependency.robot:19]: `RunThisTest` should be removed

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: poetry.lock
Comment (Raghul-M) [reply poetry.lock:1]: Yeah, By mistake I included it . Will remove it

File: ods_ci/run_robot_test.sh (hunk 1 above)
Comment (Raghul-M) [reply ods_ci/run_robot_test.sh:25]: Resolved

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource (hunk 2 above)
Comment (Raghul-M) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource:29]: Resolved

File: ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/430__data-science-pipelines-operator-dependency.robot (hunk 3 above)
Comment (Raghul-M) [reply ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/430__data-science-pipelines-operator-dependency.robot:19]: tag removed

File: ods_ci/run_robot_test.sh (hunk 1 above)
Comment (jstourac) [reply ods_ci/run_robot_test.sh:25]: Hi Raghul, I can see you commented that you resolved some of the concerns here, although, I don't see any change on this PR. Could you assure you have pushed your latest changes so it propagates here? Thanks! :slightly_smiling_face:

**Comment by Raghul-M** [comment]
Verified success with Jenkins Build #2367

File: ods_ci/run_robot_test.sh [hunk 4]
```diff
@@ -22,7 +22,7 @@ EMAIL_SERVER_SSL=false
 EMAIL_SERVER_UNSECURE=false
 OPEN_REPORT_IN_BROWSER=false
 REPORT_BROWSER="firefox" # Default browser to open reports in
-SUBFOLDER=false
+SUBFOLDER=false
```
Comment (jstourac) [inline ods_ci/run_robot_test.sh:25]:
```suggestion
SUBFOLDER=false
```

**Review by jgarciao — APPROVED** [review]

**Review by diegolovison — APPROVED** [review]

**Pull request merged**